src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -60,6 +60,38 @@ class LocalBackend extends Backend with LazyLogging {
 
   import LocalBackend._
 
+  private class TempFileFilter(prefix: String) extends FilenameFilter {
+    val regex = s"^$prefix.*\\.tmp$$".r
+    def accept(dir: File, name: String): Boolean = regex.findFirstIn(name) match {
+      case Some(s) => true
+      case _ => false
+    }
+  }
+
+  override def stdout(workflowId: WorkflowId, callFqn: FullyQualifiedName): WdlFile = {
+    val fqnParts = callFqn.split("\\.")
+    if (fqnParts.size < 2) {
+      throw new UnsupportedOperationException("FQN not complete")
```

Comment:
If there's going to be a `StdoutStderrException` (not 100% sure there needs to be) why doesn't this use it?